meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/stm32mp15-disco/boot.cmd

```diff
@@ -1,3 +1,5 @@
+setenv fdtfile stm32mp157c-dk2-scmi.dtb
+
```

Comment:
it would be great if the fdtfile on the boot.cmd could be inferred from the `KERNEL_DEVICETREE` to avoid having to set the same in different places

Reply:
Its only a suggestion for some next interactions, the PR LGTM.